### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by johnhwoods** [Pier-CMS/Pier3#14]
Hi JBordan.  Are you the sole remaining champion of Pier3?  It looks like it used to be awesome.  Your Metacello snippet suggests it can be loaded into Pharo 7 but it doesn't seem to work properly.   I thought it might be dead but it looks like you're still working on it?

**Comment by jborden23** [comment]
Yes, it is still working. I haven't loaded it in 7 for a while but it works well in Pharo 8.

**Comment by johnhwoods** [comment]
Thanks for getting back to me ... I've given it a go but When I try to edit a node I just get '... loading ...' in the wysiwyg editor and then it overwrites the original content with '...loading...' any idea what I'm doing wrong?

**Comment by johnhwoods** [comment]
Actually this seems to be a problem specific to the wysiwyg editor you get when you load add-ons, the plaintext editor in the Core load gives you a working edit mechanism.

**Comment by jborden23** [comment]
That is something that needs some work, as a work-around login with the admin user and pier (see http://www.myborden.com/pier/john-c-borden/updates/loading-pier-3.2-into-pharo-7).
